### DISCUSSION TITLE
Libre2 on smartwatch

### DISCUSSION THREAD

**Discussion by Zzorbaa** [NightscoutFoundation/xDrip#3618] [Q&A]
Hi! I am trying to configure my daugters watch to act as a master. From what i see it is only available with limiTTer conn via bubble. Now my question is is there going to be interference with readings from sensor and is it possible to setup conn with both mobile phone an watch at the same time? Curently we are using libre2+ oop2+xdrip+mobile. Tyvm

**Comment by PhilTU** [comment]
If you have xDrip+ for WearOS working as collector it shares data with xDrip+ on paired phone. But as you say direct to watch only works with transmitter device and OOP2 on watch.
